src/applications/mhv/secure-messaging/sass/secure-messaging.scss

```diff
@@ -251,6 +276,7 @@ va-alert button {
       left: 3px;
     }
   }
+  // Below is currently not in use... may use later
```

Comment:
If its not in use, remove it. we can always look in the git history if we need to get it back.